packages/web-frontend/src/reducers.js

```diff
@@ -818,7 +818,12 @@ function drillDown(
       return {
         ...state,
         isLoading: false,
-        levelContents: { ...state.levelContents, [action.drillDownLevel]: action.viewContent },
+        levelContents: {
+          ...state.levelContents,
+          [action.drillDownLevel]: {
+            viewContent: action.viewContent,
```

Comment:
Updated the the structure of the viewContent in the store for drillDown data so that is the same as viewContent data in other places (eg. enlargedDialog)